internal/util/debug/debug.go

```diff
@@ -123,3 +142,80 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 	s.Close()
 	l.Sugar().Info("Debug server stopped.")
 }
+
+func getArchiveHandler() http.HandlerFunc {
+	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		var (
+			scheme          string
+			u               url.URL
+			responses       = make(map[string][]byte, 2)
+			debugFilePrefix = "FerretDB"
+			debugFileName   = "debug.zip"
+			err             error
+		)
+
+		u.Path = metricsPath
+		u.Host = req.Host
+
+		if req.URL.Scheme == "" {
```

Comment:
It can't be anything other than "http"

Reply:
During testing I found that scheme was coming as "", hence this was required.
Any other suggestion?

Reply:
@AlekSi , WDYT?